### PR TITLE
Portenta X8: Upload sketch to the M4 [PC-803]

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- Add tutorial

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/tutorials/uploading-sketches-m4/content.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+![ADB upload with a terminal](assets/x8-terminal-ADB-push.png)
+
+## How It Works?
+The Portenta X8 has a service that waits for a sketch to be uploaded to a folder. If it detects changes the device will flash the M4 with the uploaded firmware.
+
+This work thanks to the following services:
+* **monitor-m4-elf-file.service**: this service monitors the directory `/tmp/arduino/m4-user-sketch.elf` and each time it detects a new file it will proceed to flash the M4 using  `openOCD` with the sketch that has been pushed.
+* **android-tools-adbd.service**: responsible of generating the needed interfaces for the different types of usb gadgets.
+* **create-docker-envfile.service**: Controls if the device has been plugged for example in some carrier and makes the info available at `/var/run/arduino_hw_info.env` and is meant to be used by the ´docker containers´.
```
Comment (sebromero) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/uploading-sketches-m4/content.md:75]:
```suggestion
```
I'd remove these. I don't understand how these services relate to the uploading / flashing process.

**Review by sebromero — CHANGES_REQUESTED** [review]
I made some small changes and left a comment. Have a look.

File: content/hardware/04.pro/boards/portenta-x8/tutorials/uploading-sketches-m4/content.md (hunk 1 above)
Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/uploading-sketches-m4/content.md:75]: Okay, maybe we can add that info on the fundamentals? 🤔 
Ill approve that, thanks.

**Review by sebromero — APPROVED** [review]
LGTM

**Pull request merged**